new/LIT7091QeneAyte.xml

```diff
@@ -0,0 +1,89 @@
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <listWit>
+                    <witness corresp="UppQ1524o"/>
+                </listWit>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p><foreign xml:lang="gez">Qǝne</foreign> without type indication in the manuscript. It has eight lines, meaning
+                    that it can be classified as a <foreign xml:lang="gez">mawaddəs qəne</foreign> according to
+                    <bibl><ptr target="bm:Guidi1900Qene"/></bibl>. 
+                    It is attributed to <persName ref="PRS14574GabraLeul" cert="high" corresp="SD">Gabra Ləʿul</persName> 
+                    in the manuscript.</p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Poetry"/>
+                    <term key="Qene"/>
+                    <term key="Mawaddes" cert="medium"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="JK" when="2024-10-04">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="LIT7091QeneAyte" passive="UppQ1524o"/>
+                    <relation name="saws:isAttributedToAuthor" active="LIT7091QeneAyte" passive="PRS14574GabraLeul"/>
+                </listRelation>
+            </div>
+            <div type="edition" xml:lang="gez">
+                <note>This edition is based on <ref type="mss" corresp="UppQ1524o"/> on fols 8r, 9r.
+                    Division into verses is indicated with ፤ .
+                    The end of the poem is indicated with ። .</note>
+            <ab>
+                <l n="1">አይ<corr>ቴ</corr>፡ ናስምክ፡ ውሉድ፡ አመጻ፡ ዘተመዝበረ፡ ልብነ፡ በግብረ፡ አመጻ፡ ነፋስ፤</l>
+                <l n="2"><sic>እምትዕይቱ፡</sic> ሰማየ፡ አመ፡ ይመጽእ፡ ንጉሥ፤</l>
+                <l n="3">በንጥረ፡ መባርቅት፡ ረዋዲ፡ ፈረስ፤</l>
+                <l n="4">ወከመ፡ አማጽያነ፡ ያጠፍዕ፡ በነገረ፡ ተስናን፡ ወባዕስ፤</l>
+                <l n="5">ህዳጠሒ፡ ንጸወን፡ በደብረ፡ ጽወን፡ ሐዳስ፤</l>
+                <l n="6">ደብረ፡ ጽዮን<choice resp="JK"><sic>ስ</sic><corr>ሰ</corr></choice>፡ እስመ፡ 
```

Comment:
I think resp should go with corr

Reply:
I would leave it as it is, with `<sic/>`